include/libkernelflinger/log.h

```diff
@@ -38,9 +38,13 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE *efi_log_file;
+EFI_FILE *g_root;
```

Comment:
This should just not exist (see another comment below).

Reply:
Already modified.